### ISSUE TITLE
DynamoDB port not shown in  lambda distributed trace details map

### ISSUE THREAD

**Issue by Blankll** [newrelic/node-newrelic-aws-sdk#147]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description
When checking the specific trace under lambda distributed trace, the DynamoDB's host port shows unknown in the service chart:
![Screen Shot 2022-09-08 at 14 47 36](https://user-images.githubusercontent.com/28639911/191316282-b65d6ccd-fe9b-4353-b62c-3f844b331974.png)

## Expected Behavior
the map shows the DynamoDB host port correctly.

## Your Environment
env: AWS-lambda
Runtime: node-16
NewRelic-AWS Lambda-Layers: `arn:aws:lambda:ap-southeast-2:451483290750:layer:NewRelicNodeJS16X:16`
dependencies:
```json
{
   "@aws-sdk/client-dynamodb": "^3.128.0",
    "@aws-sdk/lib-dynamodb": "^3.128.0",
     "apollo-datasource": "^3.3.2",
    "apollo-server-lambda": "^3.10.1",
    "@newrelic/apollo-server-plugin": "^2.0.1"
}
```

## Additional context

[TIP]:  # ( Add any other context about the problem here. For example, relevant community posts or support tickets. )

**Comment by jordigh** [comment]
Probably related to https://github.com/newrelic/newrelic-node-apollo-server-plugin/issues/215, let's see if support.newrelic.com can help.

**Comment by jmartin4563** [comment]
@Blankll fyi the fix for this has landed in v5.0.1
